gdk4/src/display.rs

```diff
@@ -25,6 +26,26 @@ pub trait DisplayExtManual: 'static {
 
     #[doc(alias = "gdk_display_put_event")]
     fn put_event<P: AsRef<Event>>(&self, event: &P);
+
+    /// Equivalent to the C macro `GDK_IS_WAYLAND_DISPLAY`
+    #[doc(alias = "GDK_IS_WAYLAND_DISPLAY")]
+    fn is_wayland(&self) -> bool;
```

Comment:
Instead of a function for each, you could also add an enum and a single function. Not sure what's better.

Reply:
That's a possibility yes. Then you would need to have similar methods in the enum at the end no?

Reply:
Why? You could do `display.backend() == gdk::Backend::Wayland` etc